swap_meet/vendor.py

```diff
@@ -1,2 +1,68 @@
+from operator import attrgetter
```

Comment:
Great job researching and implementing `attrgetter` for your `max()` logic below! 👍🏻